extensions/blocks/map/map.php

```diff
@@ -49,6 +63,8 @@ function jetpack_get_mapbox_api_key() {
  * @return string
  */
 function jetpack_map_block_load_assets( $attr, $content ) {
+	jetpack_record_mapbox_wpcom_load_event();
```

Comment:
If the goal is to "monitor the usage of the Mapbox API in case of abuses or excesses, and to identify and blacklist any possible problematic sites." (from the PR description)... shouldn't we only trigger this event if a WordPress.com key is being used?

Reply:
This is a very good point! Not sure how I've overlooked it 🤦‍♂